lib/cli/commands/getinfo.ts

```diff
@@ -121,7 +121,7 @@ const displayGetInfo = (response: GetInfoResponse.AsObject) => {
 
 export const command = 'getinfo';
 
-export const describe = 'get general info from the xud node';
+export const describe = 'get general info from your xud node';
```

Comment:
"your" usage here as well.

Reply:
Changed back to `get general info from the xud node`